test/metabase/api/channel_test.clj

```diff
@@ -58,6 +59,11 @@
                 :new-data     true}
                (t2/select-one-fn :details :model/Channel (:id channel)))))
 
+      (testing "can update channel description"
+        (mt/user-http-request :crowberto :put 200 (str "channel/" (:id channel))
+                              {:description "New description"})
+        (is (= "New description" (t2/select-one-fn :description :model/Channel (:id channel)))))
```

Comment:
How about using the `get/:id` route to fetch the description instead? I don't see any other coverage that the API now exposes this values.

Reply:
it's already exposed. I added it to the test here https://github.com/metabase/metabase/pull/45840/commits/f6b969496cb06c2e3d4fbedc99ee5410cdd63ea3